docker/docker_instructions_ov2.0.md

```diff
@@ -0,0 +1,157 @@
+# Run Docker Images For ROS2_OpenVINO_Toolkit
+
+**NOTE:**
+Below steps have been tested on **Ubuntu 20.04**.
+
+## 1. Environment Setup
+* Install docker ([guide](https://docs.docker.com/engine/install/ubuntu/))
+
+## 2. Build docker image by dockerfile
+```
+cd ~/ros2_openvino_toolkit/docker/Dockerfile
+vi ~/ros2_openvino_toolkit/docker/Dockerfile
+docker build --build-arg ROS_VERSION=<EXPECT_ROS_BASE_IMAGE> --build-arg VERSION=<EXPECT_ROS_VERSION> -t ros2_openvino_202201 .
+```
+For example:
+* Build image for ros_galactic
+```
+cd ~/ros2_openvino_toolkit/docker/Dockerfile
+vi ~/ros2_openvino_toolkit/docker/Dockerfile
+docker build --build-arg ROS_VERSION=galactic-desktop --build-arg VERSION=galactic -t ros2_galactic_openvino_202201 .
+```
+* Build image for ros_foxy
+```
+cd ~/ros2_openvino_toolkit/docker/Dockerfile
+vi ~/ros2_openvino_toolkit/docker/Dockerfile
+docker build --build-arg ROS_VERSION=foxy-desktop --build-arg VERSION=foxy -t ros2_foxy_openvino_202201 .
+```
+**Note**:Proxy may need to set, please check dockerfile before building.
```

Comment:
Can we add a new arg for Proxy setting to command "docer build xxxx"?

Reply:
Build arg of proxy has been added. Thanks.